### PR TITLE
Implement exclude pattern cli option

### PR DESCRIPTION
**Pull request by chambo-e**
Hello, 

I ran accross #1392 because we have almost exactly the same issue
We currently have a kinda big custom script allowing us to do filtering on recursive svgo processing and we were trying to migrate to the cli but this feature is missing

So I implementend a tiny option which accept an array of regex pattern that we can use to filter out file descriptors

### CODE REVIEW

File: lib/svgo/coa.js [hunk 1]
```diff
@@ -156,6 +160,9 @@ async function action(args, opts, command) {
     config.recursive = opts.recursive;
   }
 
+  // --exclude
+  config.exclude = opts.exclude ? opts.exclude.map(pattern => RegExp(pattern)) : [];
```
Comment (Tokimon) [inline lib/svgo/coa.js:164]: Don't you need to `new` the `RegExp`? (`new RegExp(pattern)`)

Comment (Tokimon) [reply lib/svgo/coa.js:164]: and maybe break the line up to make it a bit more readable?

```suggestion
  config.exclude = opts.exclude
    ? opts.exclude.map(pattern => RegExp(pattern))
    : [];
```

File: lib/svgo/coa.js [hunk 2]
```diff
@@ -291,7 +299,7 @@ function processDirectory(config, dir, files, output) {
  */
 function getFilesDescriptions(config, dir, files, output) {
   const filesInThisFolder = files
-    .filter((name) => regSVGFile.test(name))
+    .filter((name) => regSVGFile.test(name) && !config.exclude.some(regExclude => regExclude.test(name)))
```
[the remark this answers is not in the document]
Comment (chambo-e) [reply lib/svgo/coa.js]: Looks better, thanks :)

**Review by Tokimon — APPROVED** [review]

**Comment by chambo-e** [comment]
cc @TrySound @deepsweet if you have 5 min to take a look :)

**Comment by TrySound** [comment]
Thanks!

**Pull request merged**